wagtail/admin/tests/test_account_management.py

```diff
@@ -309,7 +310,10 @@ def test_change_password_view_post_password_mismatch(self):
 
         # Check that a validation error was raised
         self.assertTrue('new_password2' in response.context['form'].errors.keys())
-        self.assertTrue("The two password fields didn't match." in response.context['form'].errors['new_password2'])
+        if DJANGO_VERSION >= (3, 0):
+            self.assertTrue("The two password fields didn’t match." in response.context['form'].errors['new_password2'])
+        else:
+            self.assertTrue("The two password fields didn't match." in response.context['form'].errors['new_password2'])
```

Comment:
I might be missing something, but the two assertions are the same

Reply:
ah, no I take it back. it is the `didn’t` vs `didn't`

Reply:
Commenting from the bleachers here, but @gasman is there something inherent to Django 3.0 that is causing it or Wagtail to return a new character here? This seems strange, right?

https://unicode-table.com/en/0027/ - `'` Apostrophe
https://unicode-table.com/en/2019/ -  `’` Right Single Quotation Mark

Reply:
Commenting from the bleachers here, but @gasman is there something inherent to Django 3.0 that is causing it or Wagtail to return a new character here? I did find a perhaps related thing in the Django 3.0 [docs page](https://docs.djangoproject.com/en/dev/releases/3.0/#miscellaneous):
`django.utils.html.escape() now uses html.escape() to escape HTML. This converts ' to &#x27; instead of the previous equivalent decimal code &#39;.`

This does explain the change in the test for `wagtail/contrib/modeladmin/tests/test_page_modeladmin.py` later in this PR.

https://unicode-table.com/en/0027/ - `'` Apostrophe
https://unicode-table.com/en/2019/ -  `’` Right Single Quotation Mark <-- this is what you have in the PR right now.

Reply:
(lost my comment, but it's tied to my other comments above):
@gasman is there a change inherent to Django 3.0 that causes the changes you're making here with different Unicode characters? I found this in [Django 3.0 notes](https://docs.djangoproject.com/en/dev/releases/3.0/#miscellaneous) but that doesn't seem like it would cause this.
> [django.utils.html.escape()](https://docs.djangoproject.com/en/dev/ref/utils/#django.utils.html.escape) now uses [html.escape()](https://docs.python.org/3/library/html.html#html.escape) to escape HTML. This converts ' to &#x27; instead of the previous equivalent decimal code &#39;.

Here, you've got `’` [Right Single Quotation Mark](https://unicode-table.com/en/2019/) compared to the older and more standard `'` [Apostrophe](https://unicode-table.com/en/0027/)